kf_lib_data_ingest/common/constants.py

```diff
@@ -78,53 +106,106 @@ class REFERENCE_GENOME:
         HG19 = 'hg19'
 
     class PLATFORM:
+        GENOMICS = 'Complete Genomics'
         ILLUMINA = 'Illumina'
+        ION_TORRENT = 'Ion Torrent'
+        LS454 = 'LS454'
+        PACBIO = 'PacBio'
+        SOLID = 'SOLiD'
 
     class INSTRUMENT:
         HISEQ_X_v2_5 = 'HiSeq X v2.5'
         HISEQ_X_10 = 'HiSeq X Ten'
 
     class STRAND:
-        UNSTRANDED = 'Unstranded'
         FIRST = 'First Stranded'
         SECOND = 'Second Stranded'
-        OTHER = 'Other'
+        UNSTRANDED = 'Unstranded'
 
     class CENTER:
-        class WASHU:
-            NAME = 'Washington University'
-            KF_ID = 'SC_K52V7463'
+        class BAYLOR:
+            NAME = 'Baylor College of Medicine'
+            KF_ID = 'SC_A1JNZAZH'
+
+        class BC_CANCER_ANGECY:
+            NAME = 'British Columbia Cancer Agency Genome Sciences Centre'
+            KF_ID = 'SC_FN7NH453'
+
+        class BGI:
+            NAME = 'BGI@CHOP Genome Center'
+            KF_ID = 'SC_WWEQ9HFY'
 
         class BROAD:
             NAME = 'Broad Institute'
             KF_ID = 'SC_DGDDQVYR'
 
+        class FELINE_DIAGNOSTICS:
+            NAME = 'Feline Diagnostics LLC'
+            KF_ID = 'SC_CATTVETT'
+
         class HUDSON_ALPHA:
             NAME = 'HudsonAlpha Institute for Biotechnology'
             KF_ID = 'SC_X1N69WJM'
 
-        class BAYLOR:
-            NAME = 'Baylor College of Medicine'
-            KF_ID = 'SC_A1JNZAZH'
+        class NANT:
+            NAME = 'NantOmics'
+            KF_ID = 'SC_N1EVHSME'
+
+        class NOVOGENE:
+            NAME = 'Novogene'
+            KF_ID = 'SC_2ZBAMKK0'
+
+        class NCI:
+            NAME = 'National Cancer Institute, Khan Lab'
+            KF_ID = 'SC_F6RZ51K9'
+
+        class SIDRA:
+            NAME = 'Genomic Clinical Core at Sidra Medical and Research Center'
+            KF_ID = 'SC_KE2ASNJM'
 
         class ST_JUDE:
             NAME = 'St Jude'
             KF_ID = 'SC_1K3QGW4V'
 
+        class TGEN:
+            NAME = 'The Translational Genomics Research Institute'
+            KF_ID = 'SC_KQ9JZG3P'
+
+        class VIRTUAL:
+            NAME = 'Virtual'
+            KF_ID = 'SC_BATJDPHB'
+
+        class WASHU:
+            NAME = 'Washington University'
+            KF_ID = 'SC_K52V7463'
+
     class STRATEGY:
+        LINKED_WGS = 'Linked-Read WGS (10x Chromium)'
+        MRNA = 'miRNA-Seq'
+        RNA = 'RNA-Seq'
         WGS = 'WGS'
+        WXS = 'WXS'
 
     class ANALYTE:
         DNA = 'DNA'
         RNA = 'RNA'
+        VIRTUAL = 'Virtual'
 
 
 class STUDY:
     CANCER = 'Cancer'
     STRUCTURAL_DEFECT = 'Structural Birth Defect'
+    STRUCTURAL_DEFECT_AND_CANCER = 'Structural Birth Defect and Cancer'
 
     class STATUS:
```

Comment:
Why do we need anything besides PENDING? Will ingest lib ever change this status to anything besides PENDING?

Reply:
I tried to embrace possible use cases in the future since our data model has all those enums: https://github.com/kids-first/kf-api-dataservice/commit/dbdbec7af00cd9aef7ca1c01c117bb8787229b8a#diff-6623e14f4531dd961f714dda06c0faff

But, I've also used only `Pending` for my previous ingestion. If we only use `Pending` for ingestion and it never changes, is this field actually needed in DS?

Reply:
Yea, actually I don't think the ingest lib will every ingest a new study so we probably don't need any of these. @fiendish what do you think?

Reply:
I agree with you, but I also don't think it hurts to throw them in.